internal/tests/cli/boundary/_target_host_sources.bash

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+load _authorized_actions
+
+function set_target_host_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets set-host-sources -id $1 $hostSource
+}
+
+function add_target_host_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets add-host-sources -id $1 $hostSource
+}
+
+function remove_target_host_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets remove-host-sources -id $1 $hostSource
+}
+
+function target_has_host_source_id() {
+    local tid=$1
+    ids=$(boundary targets read -id $tid -format json | jq '.item.host_sources[].id')
```

Comment:
This line is only used in the `json` block, right? Should that be moved into there?